packages/fetchai/skills/carpark_detection/detection_database.py

```diff
@@ -129,11 +129,11 @@ def initialise_backend(self):
 
     def set_fet(self, amount, t):
         """Record how much FET we have and when we last read it from the ledger."""
-        self.execute_single_sql(
-            "INSERT OR REPLACE INTO fet_table(id, amount, last_updated) values(0, '{0}', '{1}')".format(
-                amount, t
-            )
+        command = (
+            "INSERT OR REPLACE INTO fet_table(id, amount, last_updated) values(0, ?, ?)"
         )
+        variables = (str(amount), str(t))
```

Comment:
Why we making strings from input data here? Why don't accept only strings on input?

Reply:
Because I just wanted to make as less changes as I could to the specific skill. And the way it was implemented it was converting them inside the database query.